app/src/main/java/org/oppia/app/player/audio/CellularDataDialogFragment.kt

```diff
@@ -0,0 +1,55 @@
+package org.oppia.app.player.audio
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.fragment.app.DialogFragment
+import android.widget.CheckBox
+import org.oppia.app.R
+
+/**
+ * DialogFragment that controls user preference for dialog box
```

Comment:
Nit: end with period for complete sentence.

Also, maybe something like the following would be clearer:

```DialogFragment that indicates to the user they are on cellular when trying to play an audio voiceover.```

Reply:
Updated comments to make it more descriptive.